### DISCUSSION TITLE
Crear puertos virtuales midi

### DISCUSSION THREAD

**Discussion by Veroledez** [earlephilhower/arduino-pico#470] [Ideas]
Hola, gracias por tu trabajo con este core para rp2040.

Me preguntaba si es posible definir y usar varios puertos virtuales midi,  asi cuando la conecte al pc aperezcan  como si hubiese conectado 4 controladores midi diferentes pero sólo a traves de un usb, no se si me explico bien, perdon.
Con teensy si es posible pero su precio es muy elevado, al menos en mi ciudad, con lo de un teensy compro unas  10 raspberry pi pico.

Muchas gracias, un saludo.

**Comment by Andy2No** [comment]
Si se refiere al Teensy 4.1, la relación de precios es casi la misma en el Reino Unido: un Teensy 4.1 cuesta alrededor de £ 30 GBP y un Raspberry Pi Pico cuesta alrededor de £ 3 GBP.

Supongo que hay bibliotecas escritas para Teensy que te permiten hacer eso, y tal vez podrían trasladarse a Pico, pero probablemente no sería una tarea sencilla.

Si el Teensy hace lo que necesitas, ¿quizás ahorres para uno de esos?

[English: If you're referring to the Teensy 4.1, the price ratio is about the same in the UK - one Teensy 4.1 costs around £30 GBP, and one Raspberry Pi Pico costs about £3 GBP.

I guess there are libraries written for the Teensy which let you do that, and maybe they could be ported to the Pico, but it would probably not be a simple task.

If the Teensy does what you need, maybe just save up for one of those? ]
